### ISSUE TITLE
NTR: Cardiac imaging or catheterization: trans-valvular pressure gradient (Cardiac & vascular)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7782]
New term request Cardiac imaging or catheterization: trans-valvular pressure gradient (nadea3_210827150635): Doppler, jet velocity, cardiac

**Comment by pnrobinson** [comment]
@MickeySegal
I think this needs to be valve specific and would suggest terms such as
Elevated aortic transvalvular pressure gradient
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7340661/

does this work on your side?

**Comment by MickeySegal** [comment]
That works; we've used it only for the aortic valve so I changed the name to "Cardiac imaging or catheterization: elevated aortic transvalvular pressure gradient"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000226
- label: Elevated aortic transvalvular pressure gradient
- parents: [Abnormal aortic physiology (HP:0030964)](https://hpo.jax.org/app/browse/term/HP:0030964)
- pmid: PMID:32451697
- def: The gradient (in this case, drop) in pressure across the aortic valve is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7782
- synonyms:
